docs/sdk/serverless-signing/quick-start.md

```diff
@@ -7,557 +7,188 @@ import TabItem from '@theme/TabItem';
-```
-
-:::note
-You should use **at least Node v19.9.0** for
+You should use **at least Node v19.9.0** for 
 - **crypto** support.
 - **webcrypto** library support if targeting `web`.
-:::
-
-### Client-Side Usage
-
-Within a file (in the Lit example repos it will likely be called `lit.js`), set up your Lit object.
 
-`client.connect()` will return a promise that resolves when you are connected to the Lit Network.
+If you're just getting started with Lit or development in general, we recommend taking a look at our [Starter Guides](https://github.com/LIT-Protocol/developer-guides-code/tree/master/starter-guides). These guides provide an environment for getting started with the Lit SDK.
 
-```jsx
-import { LitNetwork } from "@lit-protocol/constants";
+For Lit Actions specifically, you'llalso need these packages:
 
-const client = new LitJsSdk.LitNodeClient({
-  litNetwork: LitNetwork.Datil,
-});
+<Tabs
+defaultValue="npm"
+values={[
+{label: 'npm', value: 'npm'},
+{label: 'yarn', value: 'yarn'},
+]}>
+<TabItem value="npm">
 
-await client.connect();
+```bash
+npm install @lit-protocol/lit-node-client \
+@lit-protocol/constants \
+@lit-protocol/auth-helpers \
+ethers@v5
 ```
 
-:::note
-To avoid errors from Lit nodes due to stale `sessionSigs`, make sure to clear the local storage for `sessionSigs` before reconnecting or restarting the client. One way to do this is to disconnect the client first and then reconnect.
-:::
+</TabItem>
 
-The client listens to network state, and those listeners will keep your client running until you explicitly disconnect from the Lit network. To stop the client listeners and allow the browser to disconnect gracefully, call:
+<TabItem value="yarn">
 
-```jsx
-await client.disconnect();
+```bash
+yarn add @lit-protocol/lit-node-client \
+@lit-protocol/constants \
+@lit-protocol/auth-helpers \
+ethers@v5
 ```
 
-### Server-Side Usage
+</TabItem>
+</Tabs>
 
-In this example stub, the litNodeClient is stored in a global variable `app.locals.litNodeClient` so that it can be used throughout the server. `app.locals` is provided by **[Express](https://expressjs.com/)** for this purpose. You may have to use what your own server framework provides for this purpose, instead.
+## Walkthrough
 
-:::note
-Keep in mind that in the server-side implementation, the client class is named LitNodeClientNodeJs.
-:::
+### Connecting to the Lit Network
 
-`app.locals.litNodeClient.connect()` returns a promise that resolves when you are connected to the Lit network.
+Running a Lit Action requires you to connect to the Lit network. This can be done by creating a [LitNodeClient](https://v6-api-doc-lit-js-sdk.vercel.app/classes/lit_node_client_src.LitNodeClient.html) instance, which will connect to the Lit network and allow you to interact with the network. We will also be initializing an Ethereum wallet using the `ETHEREUM_PRIVATE_KEY` environment variable. This will be needed in generating session signatures.
```

Comment:
> Running a Lit Action requires you to connect to the Lit network. This can be done by creating a [LitNodeClient](https://v6-api-doc-lit-js-sdk.vercel.app/classes/lit_node_client_src.LitNodeClient.html) instance, which will connect to the Lit network and allow you to interact with the network. We will also be initializing an Ethereum wallet using the `ETHEREUM_PRIVATE_KEY` environment variable. This will be needed in generating session signatures.

Consider "Running a Lit Action requires an active connection to the Lit network. This can be done by initializing a [LitNodeClient](https://v6-api-doc-lit-js-sdk.vercel.app/classes/lit_node_client_src.LitNodeClient.html) instance, which will establish a connection to the Lit nodes.

We will also be initializing an Ethereum wallet using the `ETHEREUM_PRIVATE_KEY` environment variable, which is required in order to successfully generate session signatures."

Reply:
Changed to:
```
We will also be initializing an Ethereum wallet using the `ETHEREUM_PRIVATE_KEY` environment variable, which is required for generating session signatures in this example.
```

Because session signatures can be generated without an Ethereum wallet (e.g. getPkpSessionSigs)